src/main/java/com/jamieswhiteshirt/reachentityattributes/mixin/ServerPlayNetworkHandlerAttackMixin.java

```diff
@@ -0,0 +1,30 @@
+package com.jamieswhiteshirt.reachentityattributes.mixin;
+
+import com.jamieswhiteshirt.reachentityattributes.ReachEntityAttributes;
+import net.minecraft.entity.Entity;
+import net.minecraft.server.network.ServerPlayNetworkHandler;
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(targets = "net.minecraft.server.network.ServerPlayNetworkHandler$1")
+public class ServerPlayNetworkHandlerAttackMixin {
+    @SuppressWarnings("ShadowTarget")   // synthetic field "ServerPlayNetworkHandler.this"
```

Comment:
Use `Shadow#aliases` for synthetic targets

Reply:
That's pretty great, thanks for the info !